compiler/rustc_passes/src/liveness.rs

```diff
@@ -481,11 +481,10 @@ const ACC_USE: u32 = 4;
 
 struct Liveness<'a, 'tcx> {
     ir: &'a mut IrMaps<'tcx>,
-    body_owner: LocalDefId,
     typeck_results: &'a ty::TypeckResults<'tcx>,
     param_env: ty::ParamEnv<'tcx>,
     upvars: Option<&'tcx FxIndexMap<hir::HirId, hir::Upvar>>,
```

Comment:
I think we can get rid of this and use get_capture_kind_span where we need the span

Reply:
Please make this a commit of its own so that its easy to track any tests that get affected